src/checkers/inference/DefaultSlotManager.java

```diff
@@ -335,13 +377,119 @@ public SourceVariableSlot createSourceVariableSlot(AnnotationLocation location,
             int id = locationCache.get(location);
             sourceVarSlot = (SourceVariableSlot) getSlot(id);
         } else {
-            sourceVarSlot = new SourceVariableSlot(nextId(), location, type, true);
+            sourceVarSlot = new SourceVariableSlot(nextId(), location, type, defaultAnnotation, true);
             addToSlots(sourceVarSlot);
             locationCache.put(location, sourceVarSlot.getId());
         }
         return sourceVarSlot;
     }
 
+    /**
+     * Find the default annotation for this location by checking the real type factory.
+     * @param location Location to create a new {@link SourceVariableSlot}.
```

Comment:
> Parameter names are lowercase by convention. The data type starts with a lowercase letter to indicate an object rather than a class. The description begins with a lowercase letter if it is a phrase (contains no verb), or an uppercase letter if it is a sentence. End the phrase with a period only if another phrase or sentence follows it.

from https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html